app/models/patient.rb

```diff
@@ -10,6 +12,14 @@ class Patient
   field :secondary_person, type: String
 	field :secondary_phone, type: String
 
+  track_history   :on => [:name, :primary_phone, :secondary_person, :secondary_phone, :updated_by_id],
```

Comment:
Does this also track associated information in `pregnancy` or `clinic` like appointment date, what clinic they're going to, etc?